backend/src/v5/models/users.js

```diff
@@ -18,7 +18,9 @@
 const { USERS_COL, USERS_DB_NAME } = require('./users.constants');
 const { createResponseCode, templates } = require('../utils/responseCodes');
 const { generateHashString, sanitiseRegex } = require('../utils/helper/strings');
+const { createNewUserRecord } = require('../processors/users');
```

Comment:
this is cyclic dependency, files in model folder should never include something from the processor folder.

The idea is that if you need a bigger context, this logic should be happening in the processor folder.